packages/components/src/color-palette/types.ts

```diff
@@ -55,7 +55,7 @@ export type ColorPaletteProps = Pick< PaletteProps, 'onChange' > & {
 	 *
 	 * @default []
 	 */
-	colors?: ( PaletteObject | ColorObject )[];
+	colors?: PaletteObject[] | ColorObject[];
```

Comment:
This change is to reflect that the `colors` array cannot contain mixed `PaletteObject` and `ColorObject` items